proto/BUILD

```diff
@@ -1,17 +1,27 @@
 load("@rules_cc//cc:defs.bzl", "cc_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
 
-# _____ build cpp from proto _____
 cc_proto_library(
   name = "spec_cc_proto",
   visibility = ["//visibility:public"],
   deps = [":spec_proto"],
 )
 
+cc_proto_library(
+  name = "pubsub_cc_proto",
+  visibility = ["//visibility:public"],
+  deps = [":pubsub_proto"],
+)
+
 
 proto_library(
   name = "spec_proto",
   srcs = ["spec.proto"],
   deps = ["@com_google_protobuf//:any_proto",
   "@com_google_protobuf//:timestamp_proto"],
 )
+proto_library(
+  name = "pubsub_proto",
```

Comment:
Why not importing googleapis as a bazel target? This way, when PubSub team publishes changes to their protos, the changes will apply automatically.

Reply:
How should I do this? What build rule should I use? As I understand it, http_archive requires a compressed file and I'm not sure if google apis provides this.

I tried to use the git_repository rule as follows:
WORKSPACE:
```
load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
git_repository(
    name = "googleapis",
    remote = "https://github.com/googleapis/googleapis",
    commit = "8500bd52067f0842f56d81369ac36087afe617f8",
    shallow_since = "1594404949 -0700",
)
```

//proto/build:
```
cc_proto_library(
  name = "pubsub_cc_proto",
  visibility = ["//visibility:public"],
  deps = ["@googleapis//google/pubsub/v1:pubsub_proto"],
)
```

But got the following error:
```
error loading package '@googleapis//google/pubsub/v1': Unable to find package for @com_google_googleapis_imports//:imports.bzl: The repository '@com_google_googleapis_imports' could not be resolved. and referenced by '//proto:pubsub_cc_proto'
```

Reply:
Tried to use http_archive as so:
```
http_archive(
    name = "googleapis",
    urls = ["https://github.com/googleapis/googleapis/archive/master.zip"],
    sha256 = "cc1c331a082cc35889aaaa3d45c142980c9911294b790a4830f3e0912afef033",
)
```
ERROR: /usr/local/google/home/leexj/Documents/code/cloudevents-sdk-cpp/proto/BUILD:10:17: no such package '@googleapis//google/pubsub/v1': BUILD file not found in directory 'google/pubsub/v1' of external repository @googleapis. Add a BUILD file to a directory to mark it as a package. and referenced by '//proto:pubsub_cc_proto'
```

But got the error that a build file could not be found although there is a build.bazel file present.